docs/usage/trad-cluster.md

```diff
@@ -0,0 +1,18 @@
+# Traditional Clusters
+
+Traditional clusters, or simply "clusters" in this document. Are machines that you can provision and run yourself.
```

Comment:
I took your definition of Trad clusters verbatim, yes the opening should be one sentence. 👍